fiorella/src/styles/layout/slider.scss

```diff
@@ -0,0 +1,153 @@
+.slide {
+  display: none;
+}
+
+.slider-container {
+  position: relative;
+}
+
+.slider-navigator {
+  display: flex;
+  justify-content: center;
+  position: absolute;
+  top: 10px;
+  width: 100%;
+}
+
+.slider-nav-button {
+  background-color: $color-white-solid;
+  border-radius: 50%;
+  border-style: none;
+  font-family: sans-serif;
+  margin: 0 5px;
```

Comment:
Properties should be ordered background-color, border-radius, border-style, font-family, height, margin, width